docs/api.md

```diff
@@ -2351,6 +2375,34 @@ Only one trace can be active at a time per browser.
 #### tracing.stop()
 - returns: <[Promise]<[Buffer]>> Promise which resolves to buffer with trace data.
 
+### class: FileChooser
+
+[FileChooser] objects are dispatched by page via the ['page.interceptFileChooser'](#page-interceptfilechooser) method.
```

Comment:
The word dispatch is misleading, because we don't have events for this at the moment. They are returned.

Reply:
Done.